src/test/ui/feature-gates/feature-gate-nll.rs

```diff
@@ -1,20 +1,18 @@
 // There isn't a great way to test feature(nll), since it just disables migrate
-// mode and changes some error messages. We just test for migrate mode.
+// mode and changes some error messages.
+
+// FIXME(Centril): This test is probably obsolete now and `nll` should become
+// `accepted`.
```

Comment:
Probably, the `-Z` flags are probably sufficient for the remaining borrowck configuration. This can be a future PR.